test/remote/gateways/remote_fat_zebra_test.rb

```diff
@@ -177,17 +177,23 @@ def test_successful_purchase_with_metadata
   end
 
   def test_successful_purchase_with_3DS_information
-    assert response = @gateway.purchase(@amount, @credit_card, @options.merge(cavv: 'MDRjN2MxZTAxYjllNTBkNmM2MTA=', xid: 'MGVmMmNlMzI4NjAyOWU2ZDgwNTQ=', sli: '05'))
+    assert response = @gateway.purchase(@amount, @credit_card, @options.merge(three_d_secure: { cavv: 'MDRjN2MxZTAxYjllNTBkNmM2MTA=', xid: 'MGVmMmNlMzI4NjAyOWU2ZDgwNTQ=', eci: '05' }))
```

Comment:
If we're retaining the old form of passing in these fields, this should probably be a new test.

Reply:
I think that's my only quibble though; once these are tested individually, please squash into one commit and we'll be able to bring it in soon!

Reply:
Good point, i'll add a test for the old way too.